src/incetives-service/app.yaml

```diff
@@ -0,0 +1,6 @@
+runtime: nodejs10
+service: incentives
+env_variables:
```

Comment:
These keys should not be visible in the codebase. Need to utilize good security practices here.